testing/env.angular.js

```diff
@@ -2,7 +2,7 @@ const fs = require('fs');
 const path = require('path');
 const simpleGit = require('simple-git');
 
-const rimraf = require('./helpers/rimraf-async');
+const rimraf = require('../utility/rimraf-async');
```

Comment:
It should not be a part of the devextreme-cli package.

Reply:
Moved to testing folder